lvk/vulkan/VulkanClasses.cpp

```diff
@@ -3607,6 +3610,12 @@ VkShaderModule lvk::VulkanContext::createShaderModule(ShaderStage stage,
       vec4 textureBindlessCube(uint textureid, uint samplerid, vec3 uvw) {
         return texture(samplerCube(kTexturesCube[textureid], kSamplers[samplerid]), uvw);
       }
+      vec4 textureBindlessCubeLod(uint textureid, uint samplerid, vec3 uvw, float lod) {
+        return textureLod(samplerCube(kTexturesCube[textureid], kSamplers[samplerid]), uvw, lod);
+      }
+      int textureBindlessQueryLevels2D(uint textureid, uint samplerid) {
+        return textureQueryLevels(sampler2D(kTextures2D[textureid], kSamplers[samplerid]));
```

Comment:
Pls use sampler-less syntax everywhere wherever possible, see `textureBindlessSize2D()` above for reference.

Reply:
updated